# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../aop/hotfixes.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- ./aop.c.orig	2004-04-06 15:13:28.000000000 +0200
+++ ./aop.c	2004-04-14 22:20:13.847780240 +0200
@@ -21,7 +21,7 @@
 	init_pair(1, COLOR_RED, COLOR_BLACK);
 
 start:	memset(field, ' ', 25*81);
-	snprintf(tmp, 96, "/usr/local/share/aop/aop-level-%02d.txt", level);
+	snprintf(tmp, 96, "/usr/share/aop/aop-level-%02d.txt", level);
 	f = fopen(argc>=2 ? argv[level] : tmp, "r");
 	if (!f) { endwin(); printf("Can't open level file.\n"); return 1; }
 	opx=ipx=px=3, opy=ipy=py=2; op=p; p += 700000 + level*373737;
--- ./Makefile.orig	2004-03-19 15:26:46.000000000 +0100
+++ ./Makefile	2004-04-14 22:22:18.401845144 +0200
@@ -1,13 +1,12 @@
 
-LDFLAGS = -lncurses
-
 aop: aop.c
+	$(CC) aop.c -lncurses -o aop
 
 install: aop
-	install -d /usr/local/bin
-	install -d /usr/local/share/aop
-	install -m 755 aop /usr/local/bin/aop
-	install -m 644 aop-level-*.txt /usr/local/share/aop/
+	install -d /usr/bin
+	install -d /usr/share/aop
+	install -m 755 aop /usr/bin/aop
+	install -m 644 aop-level-*.txt /usr/share/aop/
 
 clean:
 	rm -f aop core* *~
